packages/frontend/src/pages/Login.tsx

```diff
@@ -79,17 +79,6 @@ const Login: FC = () => {
         return <PageSpinner />;
     }
 
-    if (health.status === 'success' && health.data?.needsSetup) {
```

Comment:
@rephus @ZeRego isn't this still useful? It forced a redirect to the `register` screen if there is no existing user in the database. 

Still makes sense to me!

Reply:
is it worth keeping the needsSetup flag just for that ?

Reply:
if we still want to keep it, I'd remate it to `hasUsers` or something more meaningful

Reply:
I don't think it makes sense the same way we show the register page even if they aren't allowed to register a new org. 🤔  Why/how would they end up in the login page before register an org ? 🤔